src/domain/core/src/entities/current_account_subject.rs

```diff
@@ -9,28 +9,30 @@
 
 use opendatafabric::AccountName;
 
+use crate::auth;
+
 /////////////////////////////////////////////////////////////////////////////////////////
 
-pub const TEST_ACCOUNT_NAME: &str = "kamu";
+pub const DEFAULT_ACCOUNT_NAME: &str = "kamu";
+pub const DEFAULT_AVATAR_URL: &str = "https://avatars.githubusercontent.com/u/50896974?s=200&v=4";
 
 /////////////////////////////////////////////////////////////////////////////////////////
 
 pub struct CurrentAccountSubject {
-    pub account_name: AccountName,
+    pub account: auth::AccountInfo,
```

Comment:
This feels very wrong to me. Auth subject should be the carrier of identity and nothing else. Definitely would not expect it to carry avatar URLs.

Why do you think it's necessary to have `AccountInfo` as part of the Subject? Wouldn't it be enough if where necessary we could resolve Subject into `AccountInfo` via `AuthenticationService`?

Reply:
I agree, it was a hasty move, reverted